No exported module RuntimeAdapterProvider

I'm trying to use local runtime with a custom database adapter, and I am getting an error that RuntimeAdapterProvider is not exported from "@assistant-ui/react".

Trying to import it directly shows the following error
```
Cannot find module '@assistant-ui/react/runtimes/adapters/RuntimeAdapterProvider' or its corresponding type declarations.ts(2307)
```
and also causes the app to fail with 
`Module not found: Can't resolve '@assistant-ui/react/runtimes/adapters/RuntimeAdapterProvider'`


I am using @assistant-ui/react version 0.10.19.


I'm seeing a similar export error with ThreadHistoryAdapter, but direct import works.
```
Cannot find module '@assistant-ui/react/runtimes/adapters/thread-history/ThreadHistoryAdapter' or its corresponding type declarations.ts(2307)
```

--- a/.changeset/fluffy-brooms-dream.md
+++ b/.changeset/fluffy-brooms-dream.md
@@ -1,0 +1,5 @@
+---
+"@assistant-ui/react": patch
+---
+
+Fix: Export missing adapter primitives

--- a/packages/react/src/runtimes/adapters/index.ts
+++ b/packages/react/src/runtimes/adapters/index.ts
@@ -2,3 +2,5 @@ export * from "./attachment";
 export * from "./feedback";
 export * from "./speech";
 export * from "./suggestion";
+export * from "./RuntimeAdapterProvider";
+export * from "./thread-history/ThreadHistoryAdapter";

